daprdocs/content/en/contributing/docs-contrib/contributing-docs.md

```diff
@@ -41,15 +41,18 @@ Style and tone conventions should be followed throughout all Dapr documentation
 
 ## Diagrams and images
 
-Diagrams and images are invaluable visual aids for documentation pages. Diagrams are kept in a [Dapr Diagrams Deck](https://github.com/dapr/docs/tree/v1.11/daprdocs/static/presentations), which includes guidance on style and icons. 
+Diagrams and images are invaluable visual aids for documentation pages. Use the diagram style and icons in the [Dapr Diagrams template deck](https://github.com/dapr/docs/tree/v1.14/daprdocs/static/presentations). 
 
-As you create diagrams for your documentation:
+The process for creating diagrams for your documentation:
 
-- Save them as high-res PNG files into the [images folder](https://github.com/dapr/docs/tree/v1.11/daprdocs/static/images).
-- Name your PNG files using the convention of a concept or building block so that they are grouped.
+1. Download the [Dapr Diagrams template deck](https://github.com/dapr/docs/tree/v1.14/daprdocs/static/presentations) to use the icons and colors.
```

Comment:
Is there any way that we can have the URL here be dynamic, otherwise this will need to be updated on each release. If not, then can keep this for now, but it will eventually go stale.

Reply:
I am referring to the v1.14 in the URL and whether this can use the latest tag? Have we done this anywhere else?

Reply:
I don't think we have... I can do a quick search though.